src/unit_tests/wazuh_db/test_wdb_global_parser.c

```diff
@@ -0,0 +1,187 @@
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "wazuh_db/wdb.h"
+
+int __wrap__mdebug1()
+{
+    return 0;
+}
+
+int __wrap__mdebug2()
+{
+    return 0;
+}
+
+int __wrap__mwarn()
+{
+    return 0;
+}
+
+int __wrap__merror()
+{
+    return 0;
+}
+
+int __wrap_wdb_open_global2()
+{
+    return mock();
```

Comment:
It is a good practice to make the return data type explicit. For doing this, you should use `return mock_type(int);`